assets/app/scripts/directives/oscObjectDescriber.js

```diff
@@ -26,8 +26,10 @@ angular.module('openshiftConsole')
       },
       link: function(scope, elem, attrs) {
         // TODO need to make these elements selectable via the keyboard
-        $(elem).on("click.oscobject", function() {
-          if (scope.resource) {
+        $(elem).on("click.oscobject", function(event) {
+          // Clicks directly on a link inside osc-object elements should simply
+          // follow the link.
+          if (scope.resource && event.target.nodeName !== 'A') {
```

Comment:
also, do the same check in the `mousemove` handler before adding the `osc-object-hover` class, so we don't indicate a click will do something it won't